### PR TITLE
Update to latest remap-istanbul for bug fix

### PR DESCRIPTION
**Pull request by mtraynham**
Per https://github.com/SitePen/remap-istanbul/issues/162, remap-istanbul had a dependency range issue with the latest 0.7.0 release of source-map and had to be updated.  I've bumped the remap-instanbul dependency to 0.10.1 to pick up that fix.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "url": "git@github.com:sshev/karma-remap-coverage.git"
   },
   "dependencies": {
-    "remap-istanbul": "0.8.4"
+    "remap-istanbul": "0.10.1"
```
Comment (paulblyth) [inline package.json]: Worth putting a `^` so we get any patch fixes of `0.10.*`?

Comment (mtraynham) [reply package.json]: @paulblyth Sounds good to me, I'll update and rebase.

**Comment by mtraynham** [comment]
Updated to use `"remap-istanbul": "^0.10"`.

**Comment by paulblyth** [comment]
@sshev any chance you could merge this PR and release?

**Comment by sshev** [comment]
@paulblyth merged and published to npm as `karma-remap-coverage@0.1.5`

**Comment by paulblyth** [comment]
Thanks @sshev and @mtraynham

**Pull request merged**